src/openms/source/CHEMISTRY/AASequence.cpp

```diff
@@ -1691,4 +1691,19 @@ namespace OpenMS
     return aas;
   }
 
+  AASequence::AASequence(const String& s) :  
+    n_term_mod_(nullptr),
+    c_term_mod_(nullptr)
+  {
+    parseString_(s, *this, true);
+  }
+
+  AASequence::AASequence(const char* s) :
+    n_term_mod_(nullptr),
+    c_term_mod_(nullptr)
+  {
+    parseString_(s, *this, true);
+  }
```

Comment:
can we put these at the top next to the default ctor?!
and ideally stay DRY by omitting
```
   n_term_mod_(nullptr),
    c_term_mod_(nullptr)
```
in all three cases (replace by ` ...n_term_mod_ = nullptr;` in header.